docs/tutorials/steps_to_train_models_with_multi_gpus_using_dp.md

```diff
@@ -0,0 +1,57 @@
+# Steps to train models with multiple gpus using DataParallel
+
+If you are interested in training deep forecasting models in parallel using multiple GPUs, please follow the instructions below.
+
+**Notes**: Currently, we only support **DataParallel**, and it is supported only on the **Sequential Backend**.
+
+
+### 1. Check the GPU devices information on your machine
+First and foremost, please ensure that there are multiple GPUs on your machine and that they are successfully detected. This can be accomplished in several ways:
+1) Using nvidia-smi command
+```
+nvidia-smi
+```
+2) Using torch.cuda.device_count()
+```
+import torch
+print("torch.cuda.device_count()")
+```
+
+### 2. Configure the GPUs for Use via the `gpus` Argument
+When initiating the running of the launching scripts, it is necessary to specify the GPUs to be utilized by setting the 'gpus' argument.
+
+python ./scripts/run_benchmark.py --config-path "rolling_forecast_config.json" --data-name-list "Electricity.csv" --strategy-args '{"horizon":96}' --model-name "time_series_library.PatchTST" --model-hyper-params '{"batch_size": 16, "d_ff": 1024, "d_model": 512, "lr": 0.001, "horizon": 96, "seq_len": 512}' --adapter "transformer_adapter"  `--gpus 0`  --num-workers 1  --timeout 60000  --save-path "Electricity/PatchTST"
```

Comment:
It should be a code block, and please avoid using backticks (``) within the code block. The following code blocks are the same.

Reply:
The document has been updated